src/inc/Multilingual_Press.php

```diff
@@ -405,20 +405,35 @@ private function run_frontend_actions() {
 
 		// frontend-hooks
 		$hreflang = new Mlp_Hreflang_Header_Output( $this->plugin_data->get( 'language_api' ) );
-		add_action(
-			'template_redirect',
-			array (
-				$hreflang,
-				'http_header'
-			)
-		);
-		add_action(
-			'wp_head',
-			array (
-				$hreflang,
-				'wp_head'
-			)
-		);
+
+		/**
+		 * Filters the used output methods for the hreflang links.
+		 * Possible values are 'header' and 'html'.
+		 *
+		 * @since 2.7.0
+		 *
+		 * @param String[] The used output methods.
```

Comment:
Lower-case `string[]`. However, I'm not really sure I like the array here. I think I'd prefer a bitmask representing identifiers referenced via class constants. The constant name could be something like `HREFLANG_TYPE`, with `1` and `2` being the value, respectively. The filter should then be `multilingualpress.hreflang_type`.

Do you know what I mean?